src/seedu/addressbook/commands/Command.java

```diff
@@ -70,4 +70,12 @@ public int getTargetIndex() {
     public void setTargetIndex(int targetIndex) {
         this.targetIndex = targetIndex;
     }
+
+    /**
+     * return true for command types that mutate the data.
+     * return false by default unless overidden by child class.
```

Comment:
Selecting this kinda default behavior is not suggested. While in this case it works, it shouldn't be the norm.
Think why?